docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/1.create-space.md

```diff
@@ -0,0 +1,81 @@
+# CREATE SPACE syntax
+
+```ngql
+CREATE SPACE <graph_space_name>
+    [(partition_num = <partition number>, replica_factor = <replica number>, charset = <charset>, collate = <collate>,
+    vid_type = FIXED_STRING(<string_number>))]
+```
+
+The `CREATE SPACE` statement creates a new graph space with the given name. A `SPACE` is a region that provides physically isolated graphs in Nebula Graph. An error occurs if the database exists.
+
+## IF NOT EXISTS
+
+You can use the `IF NOT EXISTS` keywords when creating spaces. This keyword automatically detects if the corresponding space exists. If it does not exist, a new one is created. Otherwise, no space is created.
+
+> **NOTE**: The space existence detection here only compares the space name (excluding properties).
+
+## Graph space name
+
+* **graph_space_name**
+
+    The name uniquely identifies a graph space on a host.
+
+## Customized graph space options
+
+There are four optional customized graph space options when creating a graph database:
+
+* `partition_num`
+
+    Specifies the number of partitions in each replica. The suggested number is five times of the hard disks number in the cluster. For example, if you have 3 hard disks in the cluster, we recommend that you set 15 partitions.
+
+* `replica_factor`
+
+    Specifies the number of replicas in the cluster. The default replica factor is 1. The suggested number is 3 in a production environment and 1 in a test environment. Always set the replica to an odd number for the need of quorum-based voting.
+
+* `charset`
+
+    Short for character set. A character set is a set of symbols and encodings. The default value is utf8.
+
+* `collate`
+
+    A set of rules for comparing characters in a character set. The default value is utf8_bin.
+
+* `vid_type`
+
+    To insert a vertex with a string-typed VID, set the `fixed_string` length for the maximum VID length. The default `fixed_string` is 8. The `fix_string(N)` in nGQL is similar to `varchar(N)` in SQL.
+
+If no option is given, Nebula Graph creates the graph space with the default options.
+
+## Example
+
+```ngql
+nebula> CREATE SPACE my_space_1; -- create space with default partition number and replica factor
```

Comment:
create a graph space with the default partition number, replica factor, and fixed_string(8)?